internal/common/compilation.bzl

```diff
@@ -292,6 +291,25 @@ def compile_ts(ctx,
   if not is_library:
     files += depset(tsickle_externs)
 
+  rerooted_es6_sources = []
+  es6_js_modules_roots = []
+
+  for source in es6_sources:
+    root = source.dirname
+    es6_js_modules_roots.append(root)
+
+    file_path = "%s/%s" % (root, source.basename.replace(".closure", ""))
```

Comment:
do we still produce the `foo.closure.js` outputs too?
I expected to see a change around https://github.com/bazelbuild/rules_typescript/blob/master/examples/es6_output/es6_output_test.sh#L5